node/packages/parser/src/ethereum/index.ts

```diff
@@ -79,6 +76,58 @@ export class TransactionParser {
       value: tx.value,
     }
 
+    const pTxWithTransfers = this.getParsedTxWithTransfers(tx, pTx, address, internalTxs)
```

Comment:
I know many have OCD about single-letter variables, I think they'd get the same sentiment here 😁 
```suggestion
    const parsedTxWithTransfers = this.getParsedTxWithTransfers(tx, pTx, address, internalTxs)
```

Reply:
Same goes for other `ptx` references that would be `parsedTx`

Reply:
😅 I hate them too - I'll get them fixed up.

Reply:
I'm actually going to leave that out of this PR, as the "pTx' convention is currently used widely, and would touch more files than I planned to in this PR.

I'll do a follow-up 👌

Reply:
I think I was doing some extra golang dev at the time and had short var names in mind haha. Feel free to update for more clarity in a follow up PR!